CHANGES.md

```diff
@@ -7,6 +7,9 @@ Unreleased
 - Speed up rule generation for libraries and executables with many modules
   (#7187, @jchavarri)
 
+- Add `--exclude-from-watch` to build options and `(watch_exclusions)`
```

Comment:
I think it is better if the command-line-option is derived mechanically from the stanza name, so if `watch_exclusions` is used as the name of the stanza, the command-line option should be called `--watch-exclusions`.

Reply:
Will change, although somewhat moot now.